test/framework/src/main/java/org/elasticsearch/test/AbstractXContentTestCase.java

```diff
@@ -161,6 +175,27 @@ public void test() throws IOException {
         }
     }
 
+    public static <T extends ToXContent> void testFromXContent(
```

Comment:
Do you think we should just call `xContentTester....test` instead of using this method?

Reply:
++